README.md

```diff
@@ -436,6 +436,15 @@ client.Subscribe(streamQuery, fauna.EventCursor("abc2345=="))
 | `fauna.StartTime`  | Sets the stream start time. Accepts an `int64` representing the start time in microseconds since the Unix epoch.<br><br>The start time must be later than the creation time of the stream token. The period between the stream restart and the start time argument can't exceed the `history_days` value for source set's collection. If a collection's `history_days` is `0` or unset, the period can't exceed 15 minutes. |
 | `fauna.EventCursor`  | Resumes the stream after the given event cursor. Accepts a `string` representation of the cursor retrieved from a `fauna.Event`. |
 
+## Debug Mode
+
+To enable debug logging, set the `FAUNA_DEBUG` environment variable to an integer for the value of the desired [slog level](https://pkg.go.dev/log/slog#Level).
+
+- `slog.LevelInfo` will log all HTTP responses from Fauna.
+- `slog.LevelDebug` will not redact the Authorization header and include the request body.
```

Comment:
Since Go doesn't provide a level finer than Debug, we should always redact the auth header.